woodwork/tests/accessor/test_statistics.py

```diff
@@ -1369,7 +1369,8 @@ def test_describe_callback(describe_df, mock_callback):
     assert mock_callback.total_elapsed_time > 0
 
 
-def test_describe_dict_extra_stats(describe_df):
+@pytest.mark.parametrize("use_age", [True, False])
+def test_describe_dict_extra_stats(use_age, describe_df):
```

Comment:
I wanted to make sure to cover age usage and confirm its behavior matches int and double usage, but I recognize that this may be a bit overkill. If anyone would rather I make a completely separate test that more minimally tests age vs integer usage, lmk.